src/components/Footer.tsx

```diff
@@ -0,0 +1,63 @@
+import { FC, useState, useEffect } from 'react';
+import { Todo } from '../types/Todo';
+import { TodosFilter } from './TodosFilter';
+
+type Props = {
+  todos: Todo[],
+  onTodoDelete: (todoIds: number[]) => void;
+};
+
+export const Footer: FC<Props> = ({ todos, onTodoDelete }) => {
+  const [activeTodosCount, setActiveTodosCount] = useState(0);
+  const [completedTodosCount, setCompletedTodosCount] = useState(0);
+  const [completedTodos, setCompletedTodos] = useState<Todo[]>([]);
```

Comment:
You don't need to create 2 separate states for it. You can calculate completedTodosCount from the completedTodos value.